python/interpret_community/common/explanation_utils.py

```diff
@@ -409,3 +410,33 @@ def _get_feature_map_from_list_of_indexes(indices_list):
             feature_map[i, j] = weights[j]
 
     return feature_map
+
+
+def _serialize_json_safe(o):
```

Comment:
My only concern is that explanation_utils could become unwieldy just like explanation_client.
A nice logical partition for these utilities might be explanation_utils that handle explanations and explanation_data_utils that handle transformations on pure data, no knowledge of importances or the explanation class hierarchy.
What do you think @gaugup ?

Reply:
or just make it something more specific, like serialization_utils.py